platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -272,6 +273,11 @@ open class GodotEditor : GodotActivity() {
 	 */
 	protected open fun enablePanAndScaleGestures() =
 		java.lang.Boolean.parseBoolean(GodotLib.getEditorSetting("interface/touchscreen/enable_pan_and_scale_gestures"))
+	/**
+	 * Define which axis of mouse wheel event will be used for mapping RotaryInput.
+	 */	
+	protected open fun getRotaryInputAxis() =
+		GodotLib.getEditorSetting("input_devices/pointing/android/rotary_input_scroll_axis")
```

Comment:
`GodotLib.getEditorSetting(...)` returns a `String`, so you need to use `Integer.parseInt(...)` to parse it to an `int` value.